api_schema/schema.go

```diff
@@ -144,6 +161,8 @@ type GetSpecResponse struct {
 	// IDs are included.
 	LearnSessionIDs []akid.LearnSessionID `json:"learn_session_ids,omitempty"`
 
+    Name string `json:"name"`
```

Comment:
nit: indentation is off, make sure your editor uses actual tabs rather than spaces (it's a go thing)

Reply:
Thanks, fixed.